tests/test_repos_freeze.py

```diff
@@ -375,3 +383,114 @@ def test_freeze_mixed(self, app):
 
         for nexp_freezed in not_expected_freezed:
             assert os.path.exists(nexp_freezed)
+
+    def test_freeze_excluded_repo(self, app):
```

Comment:
the test name is not super explicit, you're testing that freeze_age is working?

Reply:
I'm testing that we cannot freeze repos where "freezable" is set to false